.circleci/config.yml

```diff
@@ -95,7 +95,7 @@ jobs:
       - checkout
       - install-system-dependencies
       - run:
-          command: sudo apt-get install -y ffmpeg imagemagick libpam-dev
+          command: sudo apt-get install -y ffmpeg imagemagick libmagickcore-dev libmagickwand-dev libjpeg-dev libpng-dev libtiff-dev libwebp-dev libpam-dev
```

Comment:
I don't actually know, but I'll try removing it and see what it does to the CI.  I pulled those from the original PR because it seemed important, but I'll verify if they actually are. 😃

Reply:
@nightpool looks like CI fails without this change (see the commits below) 😅